[cert-manager][docs] Add information about using your own CA (#6612)
deckhouse/deckhouse: modules/101-cert-manager/docs/FAQ.md

  EOF
  ```

> ## Как использовать свой или промежуточный CA для заказа сертификатов?
> 
> Для использования собственного или промежуточного CA:
> 
> - Сгенерируйте сертификат (при необходимости):

Reviewer: ```suggestion
## How do I use a custom or interim CA to issue certificates?

Follow the steps below to use a custom or interim CA:

- Generate a certificate (if necessary):
```

  ```shell
  openssl genrsa -out rootCAKey.pem 2048
  openssl req -x509 -sha256 -new -nodes -key rootCAKey.pem -days 3650 -out rootCACert.pem
  ```

> - В пространстве имён `d8-cert-manager` создайте секрет, содержащий данные файлов сертификатов.
> 
>   Пример создания секрета с помощью команды kubectl:

Reviewer: ```suggestion
- In the `d8-cert-manager` namespace, create a secret containing certificate file data.

  An example of creating a secret with kubectl:
```

  ```shell
  kubectl create secret tls internal-ca-key-pair -n d8-cert-manager --key="rootCAKey.pem" --cert="rootCACert.pem"
  ```

>   Пример создания секрета из YAML-файла (содержимое файлов сертификатов должно быть закодировано в Base64):

Reviewer: ```suggestion
  An example of creating a secret from a YAML file (the contents of the certificate files must be Base64-encoded):
```

  ```yaml
  apiVersion: v1
  data:

>     tls.crt: <результат команды `cat rootCACert.pem | base64 -w0`>
>     tls.key: <результат команды `cat rootCAKey.pem | base64 -w0`>

Reviewer: ```suggestion
    tls.crt: <OUTPUT OF `cat rootCACert.pem | base64 -w0`>
    tls.key: <OUTPUT OF `cat rootCAKey.pem | base64 -w0`>
```

  kind: Secret
  metadata:
    name: internal-ca-key-pair
    namespace: d8-cert-manager
  type: Opaque
  ```

>   Имя секрета может быть любым.
> 
> - Создайте ClusterIssuer из созданного секрета:

Reviewer: ```suggestion
  You can use any name you like for the secret.

- Create a ClusterIssuer using the secret you created earlier:
```

  ```yaml
  apiVersion: cert-manager.io/v1
  kind: ClusterIssuer
  metadata:
    name: inter-ca
  spec:
    ca:

>       secretName: internal-ca-key-pair    # Имя созданного секрета.

Reviewer: ```suggestion
      secretName: internal-ca-key-pair    # Name of the secret you created earlier.
```

  ```

>   Имя ClusterIssuer также может быть любым.
> 
> Теперь можно использовать созданный ClusterIssuer для получения сертификатов для всех компонентов Deckhouse или конкретного компонента.
> 
> Например, чтобы использовать ClusterIssuer для получения сертификатов для всех компонентов Deckhouse, укажите его имя в глобальном параметре [clusterIssuerName](../../deckhouse-configure-global.html#parameters-modules-https-certmanager-clusterissuername) (`kubectl edit mc global`):

Reviewer: ```suggestion
  You can use any name as your ClusterIssuer name.

You can now use the created ClusterIssuer to issue certificates for all Deckhouse components or a particular component.

For example, to issue certificates for all Deckhouse components, specify the ClusterIssuer name in the [clusterIssuerName](../../deckhouse-configure-global.html#parameters-modules-https-certmanager-clusterissuername) global parameter (`kubectl edit mc global`):
```
